update: host requirements (#4549)
JetBrains/kotlin-web-site: docs/topics/multiplatform/multiplatform-publish-lib.md

## Host requirements

Kotlin/Native supports cross-compilation, allowing any host to produce necessary `.klib` artifacts.

To produce artifacts for projects with Apple targets, you'd normally need an Apple machine.
However, if you want to use other hosts, you can set this [Experimental](components-stability.md#stability-levels-explained)
option in your `gradle.properties` file:

```none
kotlin.native.enableKlibsCrossCompilation=true
```

> To build [final binaries](multiplatform-build-native-binaries.md) for Apple targets, you still need to use a Mac machine.
>
{style="note"}

> To avoid any issues during publication, publish all artifacts from one host only to avoid duplicating publications in the

Reviewer: ```suggestion
To avoid any issues during publication, publish all artifacts from a single host to avoid duplicating publications in the
```
